src/state_transition/epoch/process_pending_consolidations.zig

```diff
@@ -52,7 +52,11 @@ pub fn processPendingConsolidations(allocator: Allocator, cached_state: *CachedB
 
     if (next_pending_consolidation > 0) {
         const new_pending_consolidations = pending_consolidations.items[next_pending_consolidation..];
+        // cannot use memcpy due to overlap
```

Comment:
can we use `std.mem.copyForwards`?

Reply:
[7f4574c](https://github.com/ChainSafe/state-transition-z/pull/75/commits/7f4574c54dafadc2df9b30b98841ccd90e8ba47c) addressed here